src/config/serverConfigDefaults.ts

```diff
@@ -90,21 +90,40 @@ const ServerConfigDefaults: Partial<IServerConfig> = {
         tableAttrs: ['SAMPLE_CANCER_TYPE', 'SAMPLE_CANCER_TYPE_DETAILED'],
         priority: {
             "SAMPLE_CANCER_TYPE": 3000,
+            "PATIENT_CANCER_TYPE": 3000,
```

Comment:
Please ignore these duplicate default settings. These will be cleaned up once the backend does not require to define which group(sample/patient) the clinical attribute in. For now, we need to define them separately due the the same attribute could be defined as patient/sample differently on study level